### ISSUE TITLE
blank Rmd?

### ISSUE THREAD

**Issue by lwaldron** [LiNk-NY/Rnw2Rmd#1]
Not sure how common this is, but this was the first conversion I tried:

``` r
download.file("https://raw.githubusercontent.com/vegandevs/vegan/master/vignettes/diversity-vegan.Rnw", destfile="diversity-vegan.Rnw")
library(Rnw2Rmd)
Rnw2Rmd(from = "diversity-vegan.Rnw", to ="diversity-vegan.Rmd", validate = FALSE)
readLines(file("diversity-vegan.Rnw")) |> length()
#> [1] 796
readLines(file("diversity-vegan.Rmd")) |> length()
#> [1] 0
devtools::session_info()
#> ─ Session info ───────────────────────────────────────────────────────────────
#>  setting  value
#>  version  R version 4.2.2 Patched (2022-11-10 r83330)
#>  os       Ubuntu 20.04.5 LTS
#>  system   x86_64, linux-gnu
#>  ui       X11
#>  language (EN)
#>  collate  en_US.UTF-8
#>  ctype    en_US.UTF-8
#>  tz       America/New_York
#>  date     2023-03-15
#>  pandoc   2.18 @ /usr/lib/rstudio-server/bin/quarto/bin/tools/ (via rmarkdown)
#> 
#> ─ Packages ───────────────────────────────────────────────────────────────────
#>  package     * version date (UTC) lib source
#>  base64enc     0.1-3   2015-07-28 [1] RSPM (R 4.2.0)
#>  cachem        1.0.7   2023-02-24 [2] CRAN (R 4.2.2)
#>  callr         3.7.3   2022-11-02 [2] CRAN (R 4.2.2)
#>  cli           3.6.0   2023-01-09 [1] CRAN (R 4.2.2)
#>  colorspace    2.1-0   2023-01-23 [1] CRAN (R 4.2.2)
#>  crayon        1.5.2   2022-09-29 [2] CRAN (R 4.2.1)
#>  devtools      2.4.5   2022-10-11 [2] CRAN (R 4.2.1)
#>  digest        0.6.31  2022-12-11 [2] CRAN (R 4.2.2)
#>  dplyr         1.1.0   2023-01-29 [1] CRAN (R 4.2.2)
#>  ellipsis      0.3.2   2021-04-29 [2] CRAN (R 4.1.2)
#>  evaluate      0.20    2023-01-17 [2] CRAN (R 4.2.2)
#>  fansi         1.0.4   2023-01-22 [2] CRAN (R 4.2.2)
#>  fastmap       1.1.1   2023-02-24 [2] CRAN (R 4.2.2)
#>  fs            1.6.1   2023-02-06 [2] CRAN (R 4.2.2)
#>  generics      0.1.3   2022-07-05 [1] CRAN (R 4.2.1)
#>  ggplot2       3.4.1   2023-02-10 [1] CRAN (R 4.2.2)
#>  glue          1.6.2   2022-02-24 [2] CRAN (R 4.1.2)
#>  gtable        0.3.1   2022-09-01 [1] CRAN (R 4.2.1)
#>  htmltools     0.5.4   2022-12-07 [1] CRAN (R 4.2.2)
#>  htmlwidgets   1.6.1   2023-01-07 [1] CRAN (R 4.2.2)
#>  httpuv        1.6.9   2023-02-14 [1] CRAN (R 4.2.2)
#>  jsonlite      1.8.4   2022-12-06 [2] CRAN (R 4.2.2)
#>  knitr         1.42    2023-01-25 [2] CRAN (R 4.2.2)
#>  later         1.3.0   2021-08-18 [1] RSPM (R 4.2.0)
#>  lifecycle     1.0.3   2022-10-07 [1] CRAN (R 4.2.1)
#>  magrittr      2.0.3   2022-03-30 [2] CRAN (R 4.1.3)
#>  mdsr          0.2.7   2023-01-06 [2] CRAN (R 4.2.2)
#>  memoise       2.0.1   2021-11-26 [2] CRAN (R 4.1.2)
#>  mime          0.12    2021-09-28 [2] CRAN (R 4.1.2)
#>  miniUI        0.1.1.1 2018-05-18 [2] CRAN (R 4.1.2)
#>  munsell       0.5.0   2018-06-12 [1] RSPM (R 4.2.0)
#>  pillar        1.8.1   2022-08-19 [2] CRAN (R 4.2.1)
#>  pkgbuild      1.4.0   2022-11-27 [2] CRAN (R 4.2.2)
#>  pkgconfig     2.0.3   2019-09-22 [2] CRAN (R 4.1.2)
#>  pkgload       1.3.2   2022-11-16 [2] CRAN (R 4.2.2)
#>  prettyunits   1.1.1   2020-01-24 [2] CRAN (R 4.1.2)
#>  processx      3.8.0   2022-10-26 [2] CRAN (R 4.2.1)
#>  profvis       0.3.7   2020-11-02 [2] CRAN (R 4.2.1)
#>  promises      1.2.0.1 2021-02-11 [1] RSPM (R 4.2.0)
#>  ps            1.7.2   2022-10-26 [2] CRAN (R 4.2.1)
#>  purrr         1.0.1   2023-01-10 [2] CRAN (R 4.2.2)
#>  R.cache       0.16.0  2022-07-21 [2] CRAN (R 4.2.1)
#>  R.methodsS3   1.8.2   2022-06-13 [2] CRAN (R 4.2.0)
#>  R.oo          1.25.0  2022-06-12 [2] CRAN (R 4.2.0)
#>  R.utils       2.12.2  2022-11-11 [2] CRAN (R 4.2.2)
#>  R6            2.5.1   2021-08-19 [2] CRAN (R 4.1.2)
#>  Rcpp          1.0.10  2023-01-22 [1] CRAN (R 4.2.2)
#>  remotes       2.4.2   2021-11-30 [2] CRAN (R 4.2.2)
#>  repr          1.1.6   2023-01-26 [2] CRAN (R 4.2.2)
#>  reprex        2.0.2   2022-08-17 [1] CRAN (R 4.2.1)
#>  rlang         1.1.0   2023-03-14 [2] CRAN (R 4.2.2)
#>  rmarkdown     2.20    2023-01-19 [1] CRAN (R 4.2.2)
#>  Rnw2Rmd     * 0.99.0  2023-03-15 [1] Github (LiNk-NY/Rnw2Rmd@190ccb6)
#>  rstudioapi    0.14    2022-08-22 [2] CRAN (R 4.2.1)
#>  scales        1.2.1   2022-08-20 [1] CRAN (R 4.2.1)
#>  sessioninfo   1.2.2   2021-12-06 [2] CRAN (R 4.1.2)
#>  shiny         1.7.4   2022-12-15 [1] CRAN (R 4.2.2)
#>  skimr         2.1.5   2022-12-23 [2] CRAN (R 4.2.2)
#>  stringi       1.7.12  2023-01-11 [2] CRAN (R 4.2.2)
#>  stringr       1.5.0   2022-12-02 [2] CRAN (R 4.2.2)
#>  styler        1.9.1   2023-03-04 [2] CRAN (R 4.2.2)
#>  tibble        3.2.0   2023-03-08 [2] CRAN (R 4.2.2)
#>  tidyselect    1.2.0   2022-10-10 [1] CRAN (R 4.2.1)
#>  urlchecker    1.0.1   2021-11-30 [2] CRAN (R 4.2.1)
#>  usethis       2.1.6   2022-05-25 [2] CRAN (R 4.2.0)
#>  utf8          1.2.3   2023-01-31 [2] CRAN (R 4.2.2)
#>  vctrs         0.5.2   2023-01-23 [2] CRAN (R 4.2.2)
#>  whisker       0.4.1   2022-12-05 [2] CRAN (R 4.2.2)
#>  withr         2.5.0   2022-03-03 [2] CRAN (R 4.1.2)
#>  xfun          0.37    2023-01-31 [2] CRAN (R 4.2.2)
#>  xtable        1.8-4   2019-04-21 [1] RSPM (R 4.2.0)
#>  yaml          2.3.7   2023-01-23 [2] CRAN (R 4.2.2)
#> 
#>  [1] /mnt/STORE1/bighome/levi/R/x86_64-pc-linux-gnu-library/4.2
#>  [2] /mnt/STORE1/lib/R/bioc-release
#>  [3] /usr/lib/R/site-library
#>  [4] /usr/lib/R/library
#> 
#> ──────────────────────────────────────────────────────────────────────────────
```

<sup>Created on 2023-03-15 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>

**Comment by LiNk-NY** [comment]
Hi Levi, @lwaldron 
It should be fixed with 3c543b2af8cd0c098ce3f6e60bf64a585665a857 and 091bd57e85df60c84fe974ece7e23396b38bc85a. Note that I still have to incorporate Martin's [PR#2](https://github.com/LiNk-NY/Rnw2Rmd/pull/2) into `Rnw2Rmd`. 
Thanks!

**Comment by lwaldron** [comment]
Thanks Marcel!
